src/components/ChartSetting.jsx

```diff
@@ -6,7 +6,8 @@ import {
     SettingIcon,
     PositionLeftIcon,
     PositionBottomIcon,
-    ChevronRightIcon,
+    ThemeDarkIcon,
+    ThemeLightIcon,
```

Comment:
We don't use `ThemeDarkIcon` and `ThemeLightIcon` anymore right? It should be safe to remove from icons.jsx